meta_definitions/__init__.py

```diff
@@ -23,14 +24,14 @@ def get_all_metrics_from_file(metrics_file):
     ]
 
 
-METAS_AREA = get_all_metrics_from_file(area_metrics)
+AREA_METAS = get_all_metrics_from_file(area_metrics)
 
-METAS_AREA_LIVE_CALCULATED = get_all_metrics_from_file(area_live_calculated_metas)
+AREA_METRICS_LIVE_CALCULATED = get_all_metrics_from_file(area_live_calculated_metas)
 
-METAS_DEVICE = get_all_metrics_from_file(device_metrics)
+DEVICE_COMPLEX_METRICS = get_all_metrics_from_file(complex_device_metrics)
```

Comment:
to help the autocomplete when looking at metrics: 
```suggestion
DEVICE_METRICS_COMPLEX = get_all_metrics_from_file(complex_device_metrics)
```